doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
```

Comment:
"Using YUM repositories" should be added to:

* The administration manual index, at doc/manuals/admin/README.md
* The mkdocs.yml file, to be included in the RTD automatic rendering

The place should be just after the "Installing Orion"

Reply:
fixed in ce41e406315343b6c61358ad41eda2854b4a1b63